project_stock/models/stock_move.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import _, api, exceptions, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    task_id = fields.Many2one(
+        comodel_name="project.task",
+        string="Task for finished products",
```

Comment:
There's no finished nor raw here. Only moves. The analogy to `mrp` module was just for one part, not literally.